client/src/components/ScoresInfo/index.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react';
+
+class ScoresInfo extends Component {
+	state = {};
+	render() {
+		return <h6>Scores </h6>;
```

Comment:
you can add static data for scores, then replace it with data from the database.

Reply:
Done